numalogic/udfs/inference.py

```diff
@@ -161,6 +196,15 @@ def exec(self, keys: list[str], datum: Datum) -> Messages:
             payload.uuid,
             time.perf_counter() - _start_time,
         )
+        MODEL_STATUS_COUNTER.increment_counter(
+            payload.status.value,
+            self.__class__.__name__,
+            ":".join(payload.composite_keys),
+            payload.config_id,
+        )
```

Comment:
Can you create a method that takes in these vars along with the right Counter object that does this?

Reply:
Makes sense